tests/usbredir_test.go

```diff
@@ -105,14 +107,34 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 		It("Should fail when limit is reached", func() {
 			var tests [v1.UsbClientPassthroughMaxNumberOf + 1]testCase
 			for i := 0; i <= v1.UsbClientPassthroughMaxNumberOf; i++ {
-				ctx, cancelFn := context.WithCancel(context.Background())
-				tests[i] = testCase{
-					cancel: cancelFn,
-					err:    make(chan error),
+			retry_loop:
```

Comment:
I don't think this is required, `break` always refers to the closer loop

Reply:
It does, I actually realized afterwards so it failed (by breaking the `switch`) and I decided to use the label instead of a variable